python/lex_tools.i

```diff
@@ -50,8 +54,6 @@ public:
 class LRX: public LRXProcessor
```

Comment:
Maybe call this `LRXProc`? Also, have you made the corresponding changes to the python lib for this change and the lttoolbox change?

Reply:
I'll update these changes

Reply:
Sounds good. It might take a bit more effort than expected since you'll have to create classes for things that didn't previously exist.

Reply:
Are you referring to the weighted-automata in lttoolbox?

Reply:
No. But I realized you don't really need new classes :)